python3/amplify/README.md

```diff
@@ -0,0 +1,104 @@
+| AMPLIFY_DOMAIN (Required)                      | Amplify domain URL can find in amplify adming dashboard in section **General** inside **Production branch URL**                                                                                                                                                                                                                                             |
+| TYPE (Default: `logzio_amplify_access_lambda`) | The log type you'll use with this Lambda. This can be a [type that supports default parsing](https://docs.logz.io/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You'll need to create a new Lambda for each log type you use.                                                                                                |
+| AMPLIFY_APP_ID (Required)                      | You can find app id inside your amplify admin dashboard in section general isnide you App ARN field arn:aws:amplify:`REGION`:`AWS_ID`:apps/`APP_ID`                                                                                                                                                                                                         |
+| TIMEOUT                                        | For trigger lambda function we define (in minutes) timeout to fetch Amplify logs for range time of TIMEOUT                                                                                                                                                                                                                                                  |
+
+#### 4. Set the EventBridge (CloudWatch Events) trigger
+
+1. Find the **Add triggers** list (left side of the Designer panel) and choose **EventBridge (CloudWatch Events)** from this list.
+2. If you don't have pre defined schedule type(ex 1min) In the **Rule** click on create new rule.
+3. **Rule name** enter a name to uniquely identify your rule.
+4. **Rule description** (Optional) Provide an optional description for your rule.
+
+5. **Rule type** choose Schedule expression that suppose to be equal to the TIMEOUT of enviroment variable (ex. rate(5 minutes))
+
+#### 5. Update Permissions for Lambda Function
+
+1. Go to **Configuration** in your Lamnda function find there tab **Permissions**
+2. Click on A role name see in example **lambda-basic** it will redirect you to the IAM> Roles> lambda-basic
+   ![Lambda-role](img/lambda-role.png)
+
+3. On a role page inside Permissions tabs you will see dropdown **Add permissions** and click on **Create inline policy** it will redirect you to Create Policy page
```

Comment:
```suggestion
3. On the role page inside the **Permissions** tab, select the dropdown **Add permissions** and click on **Create inline policy**. It will redirect you to the **Create Policy** page.
```